src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -51,14 +66,20 @@ const createUiSchemaForString = (ionFormField, remediationForm, transformedResp,
     Object.assign(uiSchema, getCaptchaUiSchema());
   }
 
-  if (Array.isArray(ionFormField.options)) {
+  if(ionFormField.name === 'userProfile.countryCode'){
+    Object.assign(uiSchema, countryUISchema);
+  }
+
+  if(ionFormField.name === 'userProfile.timezone'){
+    Object.assign(uiSchema, timezoneUISchema);
+  }
+
+  if (Array.isArray(ionFormField.options) && ionFormField.options[0].value) {
```

Comment:
shall you check `ionFormField.options[0]` safely as well?

Reply:
Good question. I removed it to reduce the eslint complexity. Additionally only ionFormField.options[0].value is called? Let me know your thoughts. Thanks!

Reply:
when `ionFormField.options[0]` is undefined (in theory), `ionFormField.options[0].value` would be runtime error.
we just add ESLINT override for this file.

Reply:
Thank you very much for the suggestions! override added.